lms/djangoapps/courseware/tests/test_lti_integration.py

```diff
@@ -117,7 +116,7 @@ def mocked_sign(self, *args, **kwargs):
         self.addCleanup(patcher.stop)
 
     def test_lti_constructor(self):
-        generated_content = self.block.render(STUDENT_VIEW).content
+        generated_content = self.block.student_view(None).content
```

Comment:
This is the one thing that's holding up this PR for me at this point. Why does `self.block.render(STUDENT_VIEW).content` no longer work?

Reply:
Student view content has been updated (html was added in content), it is fully described here: https://github.com/openedx/edx-platform/pull/31134#issuecomment-1527658373
That's why I use a similar method

Reply:
Okay, I realize I'm probably being very dense here, but why was html added in the content?

My basic concern is that we've somehow changed the behavior of the `render` method, which doesn't seem like it should be necessary when removing Old Mongo functionality. If this PR does change the `render` method behavior, why does it do that? If this PR does not change the `render` method behavior, why is this change to the test necessary?

I get that HTML was added to the content, but I don't understand _why_ HTML was added to the content.

Thank you.

Reply:
@UvgenGen: Do you have time to chat about this in the next couple of days? I think maybe I'm just not understanding it properly. Please ping me on Slack when you have a moment.